Localization/es-MX/covid-19.po

```diff
@@ -82,9 +82,6 @@ msgstr "1. Donando poder de cómputo:"
 msgid "[Downloading Folding@home](/start-folding/) and helping us run simulations is the primary way to contribute. These calculations are enormous and every little bit helps! Each simulation you run is like buying a lottery ticket. The more tickets we buy, the better our chances of hitting the jackpot. Usually, your computer will never be idle, but we’ve had such an enthusiastic response to our COVID-19 work that you will see some intermittent downtime as we sprint to setup more simulations. Please be patient with us! There is a lot of valuable science to be done, and we’re getting it running as quickly as we can."
 msgstr "[Descargando Folding@home](/start-folding/) ay ayudarnos a ejecutar simulaciones es la forma principal de contribuir. ¡Estos cálculos son enormes y cada poquito ayuda! Cada simulación que ejecutes es como comprar un boleto de lotería. Cuantos más boletos compremos, mayores serán nuestras posibilidades de ganar. Por lo general, su computadora nunca estará inactiva, pero hemos tenido una respuesta tan entusiasta a nuestro trabajo de COVID-19 que verá un tiempo de inactividad intermitente a medida que avanzamos para configurar más simulaciones. ¡Por favor se paciente con nosotros! Hay mucha ciencia valiosa por hacer, y la estamos ejecutando lo más rápido posible."
 
-msgid "Don’t miss our client FAQ. [HERE](https://foldingforum.org/viewtopic.php?f=24&t=32463)"
-msgstr "No te pierdas las preguntas frecuentes de nuestro cliente. [AQUÍ] (https://foldingforum.org/viewtopic.php?f=24&t=32463)"
-
 msgid "To focus on Covid19 research set your category to any."
```

Comment:
This sentence seems to be from old source file.
`msgid` now should be `"To focus on Covid-19 research you can set the project category in your FAH client to “COVID-19”."`

Reply:
You're right. I'll close this one and make a new Pull after corrections been made.